### PR TITLE
Add some translations

### PR DESCRIPTION
**Pull request by qsomazzi**
## Changelog

``` markdown
### Added
- Added translations 
```
## Subject

Add some translations for the bundle

### CODE REVIEW

File: Resources/translations/SonataArticleBundle.en.xliff [hunk 1]
```diff
@@ -30,6 +30,54 @@
...
+                <source>breadcrumb.link_article_list</source>
+                <target>Articles</target>
+            </trans-unit>
+            <trans-unit id="list.label_title">
+                <source>list.label_title</source>
+                <target>Title</target>
+            </trans-unit>
+            <trans-unit id="list.label_updated_at">
+                <source>list.label_updated_at</source>
+                <target>Updated At</target>
```
Comment (greg0ire) [inline Resources/translations/SonataArticleBundle.en.xliff]: Why do you capitalize all words? You don't even do it for all languages, please be consistent ("Updated at")

File: Resources/translations/SonataArticleBundle.en.xliff [hunk 2]
```diff
@@ -30,6 +30,54 @@
...
+            </trans-unit>
+            <trans-unit id="list.label_status">
+                <source>list.label_status</source>
+                <target>Status</target>
+            </trans-unit>
+            <trans-unit id="help_article_title">
+                <source>help_article_title</source>
+                <target>Article's Title will be used as page Title</target>
```
Comment (greg0ire) [inline Resources/translations/SonataArticleBundle.en.xliff]: Same here, this is not german, no need for capitals here

File: Admin/ArticleAdmin.php [hunk 3]
```diff
@@ -258,7 +264,6 @@ protected function configureShowFields(ShowMapper $showMapper)
             ->add('title', 'text')
             ->add('subtitle', 'text')
             ->add('abstract', 'textarea')
-            ->add('type', 'text')
```
Comment (ChristianGripp) [inline Admin/ArticleAdmin.php]: Not related to this PR

File: Admin/ArticleAdmin.php [hunk 4]
```diff
@@ -246,6 +248,10 @@ protected function configureListFields(ListMapper $listMapper)
             ->add('updatedAt', 'datetime', array(
                 'sortable' => true,
             ))
+            ->add('status', 'choice', array(
```
Comment (ChristianGripp) [inline Admin/ArticleAdmin.php:249]: Not related to this PR

File: Admin/ArticleAdmin.php (hunk 3 above)
Comment (qsomazzi) [inline Admin/ArticleAdmin.php]: totally true, was a field that we forget to remove, you want I extract it to another PR ?

Comment (qsomazzi) [inline Admin/ArticleAdmin.php]: #14

File: Admin/ArticleAdmin.php [hunk 5]
```diff
@@ -136,13 +136,15 @@ protected function configureFormFields(FormMapper $formMapper)
...
                     'attr' => array('maxlength' => 255),
+                    'help' => $this->trans('help_article_title'),
                 ))
                 ->add('subtitle', TextType::class, array(
                     'required' => false,
                     'attr' => array('maxlength' => 255),
                 ))
                 ->add('abstract', TextareaType::class, array(
                     'required' => false,
+                    'help' => $this->trans('help_article_abstract'),
```
Comment (ChristianGripp) [inline Admin/ArticleAdmin.php]: Do we need the `trans` call? Wouldn't this be done in the twig template too?

Comment (qsomazzi) [inline Admin/ArticleAdmin.php]: There is no specific twig template to render this, it's rendered by admin form_fields. Besides it's done like this on PageBundle so I've got to admit, didn't seek much further :)

**Comment by qsomazzi** [comment]
@sonata-project/contributors all good to merge ?

File: Admin/ArticleAdmin.php (hunk 5 above)
Comment (greg0ire) [inline Admin/ArticleAdmin.php]: > seek much further

Please seek much further. How is it done in form_fields? Can you link to it? If it is not translated, why isn't it? Are there any open issues about that? I'm asking because here is what I can see : 

``` html+jinja
{% if sonata_admin.field_description.help %}
    <span class="help-block sonata-ba-field-help">{{ sonata_admin.admin.trans(sonata_admin.field_description.help, {}, sonata_admin.field_description.translationDomain)|raw }}</span>
{% endif %}
```

Something tells me you could avoid that call to `trans` (unless I'm mistaken)

Comment (qsomazzi) [inline Admin/ArticleAdmin.php]: ok then i will look into it

Comment (greg0ire) [inline Admin/ArticleAdmin.php]: Maybe you can look at the profiler to see if there is a missing translation?

Comment (qsomazzi) [inline Admin/ArticleAdmin.php]: I've got another issue with helps text that doesn't display, so I will separate this PR in 2, one for the translation, one for the help texts.
